TESTING.md

```diff
@@ -8,18 +8,22 @@ Therefore it is for the moment necessary to test `hopr-connect` with multiple pr
 
 Test setup:
 
-- bootstrap server (first node that will help as a signalling server to connect all subsequent nodes)
-- 2 or more clients
+- Charly: a bootstrap server and a relay (first node that will help as a signalling server to connect all subsequent nodes)
```

Comment:
we can remove `bootstrap server`

Reply:
Probably, but this is not the idea for this PR

Reply:
hopr-connect doesn't know about contracts and deadr, so bootstrapping of some form is needed